app/scripts/modules/core/src/pipeline/manualExecution/manualPipelineExecution.controller.js

```diff
@@ -207,7 +210,9 @@ module.exports = angular
 
     this.execute = () => {
       let selectedTrigger = this.command.trigger || {},
-        command = { trigger: selectedTrigger },
+        command = {
+          trigger: selectedTrigger,
```

Comment:
the important change

Reply:
there's no way we could've read this without it.